hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/actionhistory/ActionHistoryLayout.java

```diff
@@ -58,9 +60,16 @@ public ActionHistoryLayout(final VaadinMessageSource i18n, final DeploymentManag
         this.deploymentManagement = deploymentManagement;
         this.notification = notification;
         this.managementUIState = managementUIState;
+        actionHistoryCaption = getActionHistoryCaption("");
         init();
     }
 
+    private String getActionHistoryCaption(final String targetName) {
+        return HawkbitCommonUtil.DIV_DESCRIPTION_START
```

Comment:
Default should be "Action History" and after a target is selected "Action History for ..."